lib/danger/danger_core/plugins/dangerfile_messaging_plugin.rb

```diff
@@ -178,6 +178,8 @@ def fail(*failures, **options)
       end
     end
 
+    alias_method :failure, :fail
```

Comment:
Are you going to deprecate it eventually or `fail` will be available as-is but it's just not recommended?

Reply:
Just not recommended, no plans to remove it - when I pitch Danger Ruby is mature and not breaking anymore, I wanna live up to that.